lib/ruby/signature/environment_loader.rb

```diff
@@ -93,10 +93,6 @@ def load(env:)
 
         if stdlib_root
           signature_files.push(*each_signature(stdlib_root + "builtin"))
-          signature_files.push(*each_signature(stdlib_root + "base64"))
```

Comment:
Essentially I want to delete these lines.
I believe that loading standard library (non-builtin) signatures explicitly is better.